storagenode/console/service.go

```diff
@@ -97,117 +100,132 @@ func NewService(log *zap.Logger, consoleDB DB, bandwidth bandwidth.DB, pieceStor
+	Wallet string       `json:"wallet"`
 
-	return FromUsage(usage, s.allocatedBandwidth.Int64())
-}
+	Satellites storj.NodeIDList `json:"satellites"`
 
-// GetDailyTotalBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-// sorted in ascending order
-func (s *Service) GetDailyTotalBandwidthUsed(ctx context.Context, from, to time.Time) (_ []BandwidthUsed, err error) {
-	defer mon.Task()(&ctx)(&err)
+	DiskSpace DiskSpaceInfo `json:"diskSpace"`
+	Bandwidth BandwidthInfo `json:"bandwidth"`
 
-	return s.consoleDB.Bandwidth().GetDailyTotal(ctx, from, to)
+	Version  version.SemVer `json:"version"`
+	UpToDate bool           `json:"upToDate"`
 }
 
-// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-// sorted in ascending order for particular satellite
-func (s *Service) GetDailyBandwidthUsed(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) (_ []BandwidthUsed, err error) {
+// GetDashboardData returns stale dashboard data
+func (s *Service) GetDashboardData(ctx context.Context) (_ *Dashboard, err error) {
 	defer mon.Task()(&ctx)(&err)
+	data := new(Dashboard)
 
-	return s.consoleDB.Bandwidth().GetDaily(ctx, satelliteID, from, to)
-}
-
-// GetBandwidthBySatellite returns all info about storage node bandwidth usage by satellite
-func (s *Service) GetBandwidthBySatellite(ctx context.Context, satelliteID storj.NodeID) (_ *BandwidthInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
+	data.NodeID = s.kademlia.Local().Id
+	data.Wallet = s.walletAddress
+	data.Version = s.versionInfo.Version
+	data.UpToDate = s.version.IsAllowed()
+	data.Satellites = s.trust.GetSatellites(ctx)
 
-	summaries, err := s.bandwidthDB.SummaryBySatellite(ctx, time.Time{}, time.Now())
+	spaceUsege, err := s.pieceStore.SpaceUsedForPieces(ctx)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	// TODO: update bandwidth.DB with GetBySatellite
-	return FromUsage(summaries[satelliteID], s.allocatedBandwidth.Int64())
-}
-
-// GetUsedStorageTotal returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageTotal(ctx context.Context) (_ *DiskSpaceInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
-
-	spaceUsed, err := s.pieceStore.SpaceUsedForPieces(ctx)
+	bandwidthUsage, err := bandwidth.TotalMonthlySummary(ctx, s.bandwidthDB)
 	if err != nil {
-		return nil, err
+		return nil, SNOServiceErr.Wrap(err)
 	}
 
-	return &DiskSpaceInfo{Available: s.allocatedDiskSpace.Int64() - spaceUsed, Used: spaceUsed}, nil
-}
-
-// GetUsedStorageBySatellite returns all info about storagenode disk space usage
-func (s *Service) GetUsedStorageBySatellite(ctx context.Context, satelliteID storj.NodeID) (_ *DiskSpaceInfo, err error) {
-	defer mon.Task()(&ctx)(&err)
-
-	spaceUsed, err := s.pieceStore.SpaceUsedBySatellite(ctx, satelliteID)
-	if err != nil {
-		return nil, err
+	data.DiskSpace = DiskSpaceInfo{
+		Used:      memory.Size(spaceUsege).GB(),
```

Comment:
imho bytes better

Reply:
Typo: spaceUsage and i find the GB value good as it is!